Formula/claws-mail.rb

```diff
@@ -0,0 +1,26 @@
+class ClawsMail < Formula
+  desc "The user-friendly, lightweight, and fast email client"
+  homepage "https://www.claws-mail.org/"
+  url "https://www.claws-mail.org/releases/claws-mail-3.17.5.tar.gz"
+  sha256 "f79e4c7f089d08e6b2b3323a90b6f46991a154ce168ba4a829d593b7a5768c7b"
+
+  depends_on "pkg-config" => :build
+  depends_on "cairo"
+  depends_on "glib"
+  depends_on "gnutls"
+  depends_on "gtk+"
+  depends_on "libetpan"
+
+  def install
+    system "./configure", "--prefix=#{prefix}",
+                          "LDFLAGS=-Wl,-framework -Wl,Security",
+                          "--disable-archive-plugin",
+                          "--disable-dillo-plugin",
+                          "--disable-notification-plugin"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/claws-mail", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Okay, thanks. Let me re-work on the formula. I submitted this based off the formula for sylpheed (off which claws-mail is based).

Reply:
Here is the new test:

`assert_equal ".claws-mail", shell_output("#{bin}/claws-mail --config-dir").strip`